pieces.py

```diff
@@ -178,28 +176,25 @@ class Knight(Piece):
                             [-40, -20,  0,  5,  5,  0, -20, -40],
                             [-50, -40, -30, -30, -30, -30, -40, -50]]
 
-    def __init__(self, square, is_white):
-        image = self.BLACK_IMAGE
-        if is_white:
-            image = self.WHITE_IMAGE
-        super().__init__(image, square, is_white, 'Knight')
+    def __init__(self, square, team):
```

Comment:
If you don't do anything in _init_ method you can't just delete it, and then then init method of the base class will be called.

Reply:
fixed